lib/beaker/hypervisor/openstack.rb

```diff
@@ -197,15 +197,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        hostname = ip.ip.gsub '.','-'
-        host[:vmhostname] = hostname+'.rfc1918.puppetlabs.net'
-        @logger.debug "Provisioning #{host.name} (#{host[:vmhostname]})"
+        host[:vmhostname] = ip.ip.gsub! '.','-'
```

Comment:
@spjmurray this was changed to the non-bang version recently in [this changeset](https://github.com/puppetlabs/beaker/commit/5f50452a3b5f70e15a423449684d2033a2488805). If you check that code out, it looks like it was setting it back to '.'s rather than '-'s just before the `host[:ip]` got set on line 237, so that code didn't change the `ip.ip` value.

It looks like you'd rather keep that value as '-'s here, and seem fine setting the `host[:ip]` to use those dashes. Was something broken using the '.'s? Or am I misunderstanding this code?